package.json

```diff
@@ -24,20 +30,28 @@
   "dependencies": {
     "@thirdweb-dev/sdk": "^2.3.20",
     "@web-std/file": "^3.0.2",
+    "async-retry": "^1.3.3",
     "cbor": "^8.1.0",
     "commander": "^9.1.0",
+    "cpy": "^9.0.1",
     "enquirer": "^2.3.6",
     "ethers": "^5.6.9",
     "form-data": "^4.0.0",
+    "got": "11.8.5",
     "inquirer": "^8.2.3",
     "ipfs-unixfs": "^6.0.7",
     "ipld-dag-pb": "^0.22.3",
     "isomorphic-fetch": "^3.0.0",
     "multihashes": "^4.0.3",
     "open": "^8.4.0",
+    "prompts": "^2.4.2",
+    "rimraf": "^3.0.2",
     "solc": "0.8.13",
+    "tar": "4.4.18",
     "tslog": "^3.3.3",
+    "update-check": "^1.5.4",
     "update-notifier": "^5.1.0",
+    "validate-npm-package-name": "^4.0.0",
```

Comment:
what's this for? @nachoiacovino

Reply:
this is to validate if it's a valid npm package name :D